bundles/org.openhab.binding.networkupstools/README.md

```diff
@@ -0,0 +1,127 @@
+# Network UPS Tools Binding
+
+The primary goal of the [Network UPS Tools](http://www.networkupstools.org/) (NUT) project is to provide support for power devices, such as uninterruptible power supplies (UPS), Power Distribution Units and Solar Controllers.
+
+Network UPS Tools (NUT) provides many control and monitoring features, with a uniform control and management interface.
+More than 100 different manufacturers, and several thousands of models are compatible.
+
+This binding lets you integrate NUT servers with openHAB.
+
+## Supported Things
+
+The binding can connect to multiple NUT instances.
+The thing is a `ups` thing.
+The thing supports a number of nut features out-of-the-box.
+And supports the option to configure additional channels to get other NUT variables.
+The thing queries the NUT server for the status of the UPS approximate every 3 seconds.
+And updates the status when a change happens.
+When the UPS status changes it updates all other linked channels.
+Besides the UPS status change update the channels are updated at the user configured refresh time.
+
+Some nut variables are static in nature and are not suited for a channel.
+Some of these could be updated, like of firmware version.
+Therefore the properties are updated with a 1 hour frequency.
+The following nut variables are are read and added to the thing as properties:
+
+| Property         | Description
+|------------------|----------------------------------------
+| ups.firmware     | UPS firmware
+| ups.firmware.aux | Auxiliary device firmware
+| ups.id           | UPS system identifier
+| ups.mfr          | UPS manufacturer
+| ups.mfr.date     | UPS manufacturing date
+| ups.model        | UPS model
+| ups.serial       | UPS serial number
+
+
+## Discovery
+
+Discovery is not supported.
+
+## Thing Configuration
+
+The thing configuration requires the name of the ups device as configured with the NUT.
```

Comment:
"...as configured on the NUT server."

Because NUT has a client server architecture, making a distinction between the clients and the server adds clarity here I think.